config/admin-functions/copy-logs.sh

```diff
@@ -34,8 +33,10 @@ cp -rp /var/log/auth.log* "$DIRECTORY"
 cp -rp /var/log/vx-logs.log* "$DIRECTORY"
 
 # unmount the USB stick to make sure it's all written to disk
-pumount "$DEVICES"
+echo "Saving logs to USB drive..."
+sync $MOUNTPOINT
+sudo /vx/code/app-scripts/umount.sh
```

Comment:
Fixing a little bug that @mattroe and I have noticed where the logs may not be flushed to the disk.